recipes/write_behind/oracle/rs/README.md

```diff
@@ -33,9 +33,9 @@ select * from car;
 ORACLE=<ip> bash <(curl -fsSL https://cutt.ly/redisgears-wb-setup-oracle-node)
 ```
 * Download the Redis Gears module and add it to the cluster modules list.
-	* For Ubuntu Xenial, use [this](http://redismodules.s3.amazonaws.com/lab/08-gears-write-behind/redisgears.linux-xenial-x64.99.99.99-3e6d45a.zip).
-	* For Ubuntu Bionic, use [this](http://redismodules.s3.amazonaws.com/lab/08-gears-write-behind/redisgears.linux-bionic-x64.99.99.99-3e6d45a.zip).
-	* For RHEL7, use [this](http://redismodules.s3.amazonaws.com/lab/08-gears-write-behind/redisgears.linux-centos7-x64.99.99.99-3e6d45a.zip).
+	* For Ubuntu Xenial, use [this](http://redismodules.s3.amazonaws.com/redisgears/snapshots/redisgears.linux-xenial-x64.master.zip).
+	* For Ubuntu Bionic, use [this](http://redismodules.s3.amazonaws.com/redisgears/snapshots/redisgears.linux-bionic-x64.master.zip).
+	* For RHEL7, use [this](http://redismodules.s3.amazonaws.com/redisgears/snapshots/redisgears.linux-centos7-x64.master.zip).
```

Comment:
Not sure its a good idea to point to maste download on documentation

Reply:
We change it from time to time according to the state of the module. It's possible to pinpoint it with a git sha string. If we do a release, we'll then pinpoint it to the release, but then again, when a bug fix comes along, we'll probably move it again.

Reply:
I think we need to set it to an official release, hope to release today.